reconcile/utils/aws_api.py

```diff
@@ -609,7 +609,7 @@ def init_ecr_auth_tokens(self, accounts):
 
         self.auth_tokens = auth_tokens
 
-    def get_cluster_vpc_id(self, account, route_tables=False):
+    def get_cluster_vpc_id(self, account, route_tables=False, subnets=False):
         """
         Returns a cluster VPC ID.
```

Comment:
No longer true, it returns much more :)

Please document with an example what this function returns exactly. It will help a lot to understand the rest of the code

Since this function has grown in scope, I would rename it to something more related to what it currently does

Reply:
DONE!